### ISSUE TITLE
Does it work with `zustand@4.0.0-rc.1`? / update dependencies

### ISSUE THREAD

**Issue by tordans** [BowlingX/geschichte#31]
I testing switching from `use-query-param` to `geschichte` after introducing zustand this week (_Update: I gave up this plan…_). However, there is an issue when using the newest zustand version – see below.

It does install using `npm install geschichte --legacy-peer-deps`.

Do you know if I should expect issues when using it this way?

```
% npm install geschichte

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: radwege-check.de@0.1.0
npm ERR! Found: zustand@4.0.0-rc.1
npm ERR! node_modules/zustand
npm ERR!   zustand@"4.0.0-rc.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer zustand@">=3.6.0" from geschichte@9.1.1
npm ERR! node_modules/geschichte
npm ERR!   geschichte@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by BowlingX** [comment]
As zustand `4.x` has no stable release yet I did not try it yet. However the RC looks good and I will check if I can release a new version with `zustand` 4.x suppport.

**Comment by BowlingX** [comment]
I released version `10.0.0` which adds support for the latest version of `zustand`.
